browsergym/core/src/browsergym/core/observation.py

```diff
@@ -146,6 +146,21 @@ def extract_screenshot(page: playwright.sync_api.Page):
 
     return img
 
+def extract_mouse_position(page: playwright.sync_api.Page):
+    """
+    Extracts the mouse location on a Playwright page using a hacky JS code.
+
+    Args:
+        page: the playwright page of which to extract the mouse location.
+
+    Returns:
+        An array of the x and y coordinates of the mouse location.
+    """
+    position = page.evaluate("""() => {
```

Comment:
This will work for simple pages, but I'm worried about iframes. Here is something that could work:
 - in the JS callback (mousemove), record the position in JS in the `window` object, and also record which page / frame received this event, in Python with a method similar to `_activate_page_from_js()`.
  - to extract the mouse position in the browser viewport, take the latest mouse position (last iframe that received a mousemove event), and work your way up the frame hierarchy to reconstruct the current mouse position. See how we do that to get the coordinates of all elements in all iframes here:

https://github.com/ServiceNow/BrowserGym/blob/main/browsergym/core/src/browsergym/core/observation.py#L293-L377